tests/sce/bash_list.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+ls -al /
+RET=$?
+
+if [[ $RET != 0 ]] ; then
+    exit $XCCDF_RESULT_FAIL
+else
+    exit $XCCDF_RESULT_PASS
+fi
```

Comment:
This test  can be much simpler and not depend on the machine we are testing. Why is it listing the root dir?